example/haiku.rag.yaml

```diff
@@ -1,13 +1,11 @@
 # haiku.rag configuration file
 # See https://ggozad.github.io/haiku.rag/configuration/ for details
 
-# Configure 'production' for standard CLI logging, 'development' for Logfire
 environment: development
 
 storage:
-  data_dir: ''
-  disable_autocreate: true
-  vacuum_retention_seconds: 60
+  data_dir: ""
+  vacuum_retention_seconds: 86400
```

Comment:
Is that the `haiku-rag` slipping in by default, or do you recommend that we actually increase this value?